ci-operator/config/openshift/router/openshift-router-master.yaml

```diff
@@ -140,6 +140,26 @@ tests:
   steps:
     cluster_profile: azure4
     workflow: openshift-upgrade-azure
+- always_run: false
+  as: perfscale-aws-ingress-perf
+  optional: true
+  steps:
+    cluster_profile: aws-perfscale
+    env:
+      BASE_DOMAIN: perfscale.devcluster.openshift.com
+      COMPUTE_NODE_REPLICAS: "9"
+      COMPUTE_NODE_TYPE: m5.2xlarge
+      LOKI_USE_SERVICEMONITOR: "false"
+      OPENSHIFT_INFRA_NODE_INSTANCE_TYPE: c5.4xlarge
+      SET_ENV_BY_PLATFORM: custom
+    post:
+    - chain: ipi-aws-post
+    pre:
+    - chain: ipi-aws-pre
+    - chain: create-infra-move-ingress-monitoring-registry
+    test:
+    - chain: openshift-qe-ingress-perf
```

Comment:
Does this test chain name have anything to do with the mismatch of test names in the PR output?